tox.ini

```diff
@@ -4,7 +4,7 @@ envlist = py26, py27
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}/prestoadmin
-commands = nosetests {posargs}
+commands = nosetests --with-timer --timer-ok 60s --timer-warning 300s {posargs}
```

Comment:
did you check that this is the right time scale for our product tests?  How many end up in each category?